### DISCUSSION TITLE
Enabling a filebeat module

### DISCUSSION THREAD

**Discussion by bcmcevoy** [Security-Onion-Solutions/securityonion#1714] [Unsupported Versions]
Hello, I'm relatively new to security onion and I am trying to enable a module in filebeat to parse sonicwall logs, I can't seem to figure out how to enable the module, I can't seem to locate the filebeat.yml and synch it to elasticstack to get the module. If someone can tell me what the commands are I would appreciate it greatly.

**Comment by weslambert** [comment]
If you would like to make changes to filebeat.yml, you can copy the file from `/opt/so/saltstack/default/salt/filebeat/etc/filebeat.yml` to `/opt/so/saltstack/local/salt/filebeat/etc/filebeat.yml`.

**Comment by bcmcevoy** [comment]
Thank you so much. From what I am understanding, security onion does not come with the filebeat module installer, this is why on their website they advise you to download it. Basically, between my understanding from their website instructions and what you are telling us, once filebeat is downloaded, we enable the sonicwall module, and then perform the same steps you advised us to do with the .yml file which was created from the filebeat download. Is my understanding correct?

**Reply by weslambert** [reply]
Sounds like you will want to download the non-OSS version of Filebeat.

**Comment by halatsis1** [comment]
I have the same question! I copied the file over as advised above with the modification of adding the module I wanted but it did not install the module, just tries to access the module files, which do not exist.

**Reply by weslambert** [reply]
Sounds like you will want to download the non-OSS version of Filebeat.

**Comment by bcmcevoy** [comment]
But after you install the non-OSS Filebeat how do we enable it to interact with SO, the module folder will exist but not be in the salt stack, so how would we point SO to the module folder.

**Reply by matagatoz** [reply]
Hi where you able to enable and send the logs to SO ? im trying to acomplish the same but with the Cisco module

**Reply by thunderwood19** [reply]
> 
> 
> But after you install the non-OSS Filebeat how do we enable it to interact with SO, the module folder will exist but not be in the salt stack, so how would we point SO to the module folder.

update?

**Reply by weslambert** [reply]
You may also want to review the following documentation:
https://docs.securityonion.net/en/2.3/filebeat.html#modules
